crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D412_sections.py.snap

```diff
@@ -1,27 +1,13 @@
 ---
 source: crates/ruff_linter/src/rules/pydocstyle/mod.rs
 ---
-sections.py:216:5: D412 [*] No blank lines allowed between a section header and its content ("Short summary")
+sections.py:218:5: D412 [*] No blank lines allowed between a section header and its content ("Short summary")
     |
-214 |   @expect("D407: Missing dashed underline after section ('Raises')")
-215 |   def multiple_sections():  # noqa: D416
-216 |       """Toggle the gizmo.
-    |  _____^
-217 | | 
-218 | |     Short summary
-219 | |     -------------
-220 | | 
-221 | |     This is the function's description, which will also specify what it
-222 | |     returns.
-223 | | 
-224 | |     Returns
-225 | |     ------
-226 | |     Many many wonderful things.
-227 | |     Raises:
-228 | |     My attention.
-229 | | 
-230 | |     """
-    | |_______^ D412
+216 |     """Toggle the gizmo.
+217 | 
+218 |     Short summary
+    |     ^^^^^^^^^^^^^ D412
```

Comment:
Here the range is possibly two lines too high? It should probably be highlighting the blank line below the underline